website/views.py

```diff
@@ -366,3 +366,7 @@ def redirect_about(**kwargs):
 
 def redirect_howosfworks(**kwargs):
     return redirect('/getting-started/')
+
+
+def redirect_meetings_analytics_link(**kwargs):
+    return redirect('/getting-started/?utm_source=osf4m&utm_medium=email&utm_campaign=success')
```

Comment:
Ah, I didn't notice that the second link was slightly different! Adding second redirect now.